File: GAMEOFLIFEJAVA/gameoflife/organisms/plants/Dandelion.java
package gameoflife.organisms.plants;

import gameoflife.structs.Vector;

public class Dandelion extends Plant {

    public Dandelion(Vector position){
        super("*", 0, position);
    }

    @Override
    public void Action(){
        
        super.Action();
        super.Action();
        super.Action();
    }
    
}

File: GAMEOFLIFE/Dandelion.cpp
#include "Dandelion.h"

Dandelion::Dandelion(int x, int y): Plant('*', 8, x, y, 0,  0) {
}

void Dandelion::Action() {
	for (int i = 0; i < 3; i++) {
		Plant::Action();
	}
	SetAge(GetAge()-2);
}

Organism* Dandelion::Child() {
	return new Dandelion(*this);
}